### PR TITLE
Move singularity cross into bumped

### PR DESCRIPTION
**Pull request by stonepillars**
[OBJECTS][CHORE]

## About the PR 

Move the code that lets you sometimes go through a singularity shield into Bumped()

## Why's this needed? 

Having non-deterministic Cross sucks

### CODE REVIEW

**Comment by frawhst** [comment]
Just give it a 10% chance for it to be considered passable for pathfinding 😎

File: code/obj/machinery/singularity.dm [hunk 1]
```diff
@@ -927,15 +927,13 @@ for some reason I brought it back and tried to clean it up a bit and I regret ev
...
 	. = ..()
 	if(iscarbon(O))
-		shock(O)
-
-/obj/machinery/containment_field/Cross(atom/movable/mover)
-	. = ..()
-	if(prob(10))
-		. = TRUE
+		if(prob(10))
+			O.set_loc(get_turf(src))
```
Comment (stonepillars) [inline code/obj/machinery/singularity.dm:934]: Allows players to pass through dense objects. Working on it.

Comment (stonepillars) [reply code/obj/machinery/singularity.dm:934]: me give up

Comment (Tarmunora) [reply code/obj/machinery/singularity.dm:934]: mood

**Pull request closed without merging**